ladybug_geometry/geometry3d/cone.py

```diff
@@ -0,0 +1,191 @@
+# coding=utf-8
+"""Cone3D"""
+from __future__ import division
+
+from .pointvector import Point3D, Vector3D
+
+import math
+
+
+class Cone3D(object):
+    """Cone3D object.
+
+    Args:
+        vertex: A Point3D representing the vertex of the cone.
+        axis: A Vector3D representing the direction of the cone.
+        angle: A degree in radians representing the half angle between
+            the axis and the surface.
+        height: A number representing the height of the cone.
```

Comment:
I was under the initial impression that the magnitude of the axis vector gives you the height of the cone but I see that you have a completely separate property for height here. Am I correct in assuming that the magnitude of the axis vector is meaningless in the current setup? If so, I think we should change it and we should get rid of this height property in favor of using the vector's magnitude. The lighter and less redundant we can keep these objects's properties, the better.

Reply:
Yes, I was about to make axis normalized to be less redundant, mostly because I was not sure if it was okay to combine two cone math elements in one python element. But I agree we need to decide in favor of what keep things lighter, and I see now how it does simplify a lot of things throughout the code.